src/util/checkForMarkoCliUpdates.js

```diff
@@ -0,0 +1,15 @@
+const updateNotifier = require('update-notifier');
+const pkg = require('../../package.json');
+
+module.exports = () => {
+    const notifier = updateNotifier({
+        pkg,
+        updateCheckInterval: 0
+    });
+
+    // make call to the notifier
+    notifier.notify({
+        defer: true,
+        isGlobal: true
```

Comment:
This module is often installed local to the project as a devDependency. Thoughts on whether we should look to see if it's being used in the project and modify this flag accordingly?